.terserrc

```diff
@@ -0,0 +1,7 @@
+{
+	"mangle": false,
```

Comment:
what benefit do we have from using terser if we're not going to mangle the code?
honest question, i haven't used terser

beautify hopefully does nothing if the code passes linting, right?